lib/game.rb

```diff
@@ -1,19 +1,23 @@
 require 'mover'
 require 'board'
 require 'game_end_checker'
+require 'opponent_selector'
 
 class Game
   private
 
-  attr_reader :player, :board, :game_end_checker, :mover
+  attr_reader :board, :game_end_checker, :mover
   attr_writer :current_player
 
   public
 
+  attr_accessor :opponent, :opponent_selector
   attr_reader :current_player
 
   def initialize
+    @opponent = nil
     @current_player = 1
+    @opponent_selector = OpponentSelector.new
```

Comment:
There's no corresponding test for this, which also highlights that it's not actually used in the `Game`. I'd expect the `Game` to use the selection logic as part of this card.